package.json

```diff
@@ -6,12 +6,15 @@
   "author": "EvitaDAO team",
   "license": "MIT",
   "dependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.6",
```

Comment:
This division (dev/not dev) is not very important for us. But how I decide in which section what dependencies must be: if it could be used in prod - it is not dev dependency. hardhat ethers is used in scripts/tasks. In theory they could be used in prod . So it is not dev dependency